docs/src/app/components/raw-code/floating-action-button-code.txt

```diff
@@ -6,7 +6,7 @@
 
 <FloatingActionButton iconClassName="muidocs-icon-action-grade" secondary={true}>
```

Comment:
@shaurya947 no, the `<ToggleStar />` is the child, the closing tag is on line 11.